microsoft/testsuites/xfstests/xfstests.py

```diff
@@ -214,13 +214,17 @@ def _install_dep(self) -> None:
             isinstance(self.node.os, Redhat)
             and self.node.os.information.version < "8.0.0"
         ):
+
             if isinstance(self.node.os, Oracle):
                 posix_os.install_packages("oracle-softwarecollection-release-el7")
             else:
+                arch = self.node.os.get_kernel_information().hardware_platform
                 posix_os.install_packages(packages="centos-release-scl")
                 posix_os.install_packages(
-                    "http://mirror.centos.org/centos/7/os/x86_64/Packages/"
-                    "xfsprogs-devel-4.5.0-22.el7.x86_64.rpm"
+                    (
+                        f"http://mirror.centos.org/centos/7/os/{arch}/Packages/"
+                        f"xfsprogs-devel-4.5.0-22.el7.{arch}.rpm"
```

Comment:
there is no arm64 folder under http://mirror.centos.org/centos/7/os/

Reply:
I thought it returned aarch64 which does exist

Reply:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/10083705/215688806-75e21b2a-f8e7-4522-b5ec-e068b317f6d9.png">

Reply:
I did not realize that this was a different URL from the others. I've updated to handle this case now.